scapy/layers/tls/record.py

```diff
@@ -288,6 +294,8 @@ def dispatch_hook(cls, _pkt=None, *args, **kargs):
                     if s.rcs and not isinstance(s.rcs.cipher, Cipher_NULL):
                         from scapy.layers.tls.record_tls13 import TLS13
                         return TLS13
+        if _pkt and len(_pkt) < 5:
+            return TLSContinuousData
```

Comment:
Please use `Raw` instead. Wireshark is misleading as "TLS continuous data" does not mean anything in the TLS context.

Reply:
Updated ! Thanks

Reply:
Actually, I think `conf.raw_layer` would be more appropriate. What do you think?